source/feature/feature-template.html.md

```diff
@@ -34,7 +34,13 @@ Include you email address that you can be reached should people want to contact
 
 ### Detailed Description
 
-Expand on the summary, if appropriate. A couple sentences suffices to explain the goal, but the more details you can provide the better.
+Expand on the summary, if appropriate. Explain the goal; the more details you can provide the better. 
```

Comment:
Suggest to change to:
"Expand on the summary. Make sure the goal is clearly explained. The more detail you can provide, the better."